reminders/tests.py

```diff
@@ -72,15 +79,34 @@ def test_exist_reminder(self):
                                                         user_id__username="testUser3")
         )
 
-    def test_reminder_with_invalid_time(self, participant0, date_time=TIME_0):  # noqa: F811
+    def test_reminder_with_invalid_time(self, participant0):  # noqa: F811
         date_time = datetime(2022, 3, 22, 12, 12, 12, 0, tzinfo=timezone.utc)
-        expected = f'{time_format(date_time)} should be bigger than current.'
 
-        with pytest.raises(ValidationError, match=expected):
+        with pytest.raises(IntegrityError, match=PAST_DATE_TIME_ERROR):
             save_reminder(
                 create_reminder(participant0, METHOD_0, MESSAGES_0, date_time)
             )
 
-    def test_invalid_reminder_exist_twice(self, persist_reminder):
-        with pytest.raises(Exception, match=EXIST_REMINDER_ERROR):
-            persist_reminder.save()
+    def test_invalid_reminder_exist_twice(self):
```

Comment:
AFAIU you try to add valid reminder twice so maybe do it like:
```suggestion
    def test_reminder_exist_twice(self):
```

Reply:
done. thanks 😄